### PR TITLE
Add link to item type

### LINKED ISSUE #78: Delete action for links shows 'Delete directory'
nav-bug-links

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
Closes #78

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -238,21 +238,21 @@ const NavigatorCardBody = ({ currentFilter, files, isGrid, setPath, path, sortBy
...
               data-item={file.name} variant="plain"
               onDoubleClick={() => onDoubleClickNavigate(path, file)} onClick={() => setSelected(file)}
-              onContextMenu={(e) => { e.stopPropagation(); setSelectedContext(file) }} className={"item-button " + (file.type === "directory" ? "directory-item" : "file-item")}
+              onContextMenu={(e) => { e.stopPropagation(); setSelectedContext(file) }} className={"item-button " + (file.type.includes("directory") ? "directory-item" : "file-item")}
             >
                 <Flex direction={{ default: isGrid ? "column" : "row" }} spaceItems={{ default: isGrid ? "spaceItemsNone" : "spaceItemsMd" }}>
                     <FlexItem alignSelf={{ default: "alignSelfCenter" }}>
                         <Icon size={isGrid ? "xl" : "lg"} isInline>
-                            {file.type === "directory"
+                            {file.type.includes("directory")
```
Comment (KKoukiou) [inline src/app.jsx:255]: Why this change? there are 4 types. 'file', 'directory', 'link', 'special'
https://github.com/cockpit-project/cockpit/blob/main/src/cockpit/channels/filesystem.py#L55

**Review by KKoukiou — APPROVED** [review]

File: src/app.jsx (hunk 1 above)
Comment (MahmoudHamdy02) [reply src/app.jsx]: I've adjusted the code for the new bridge

File: src/fileActions.jsx [hunk 2]
```diff
@@ -81,9 +81,7 @@ export const editPermissions = (Dialogs, options) => {
 export const ConfirmDeletionDialog = ({ selected, itemPath, path, setHistory, setHistoryIndex }) => {
     const Dialogs = useDialogs();
 
-    const modalTitle = selected.type === "file"
-        ? cockpit.format(_("Delete file $0?"), selected.name)
-        : cockpit.format(_("Delete directory $0?"), selected.name);
+    const modalTitle = cockpit.format(_("Delete $0 $1?"), selected.type || "directory", selected.name);
```
[the remark this answers is not in the document]
Comment (MahmoudHamdy02) [reply src/fileActions.jsx:84]: If no item is selected, then `selected` contains the current directory, which doesn't have `type`. Will expand it 👍

**Review by KKoukiou — APPROVED** [review]
